content/en/agent/cluster_agent/endpointschecks.md

```diff
@@ -111,17 +137,23 @@ Enable the `endpointschecks` configuration providers on the Datadog **Node** Age
           polling: true
     ```
 
-**Note**: If the monitored endpoints are not backed by pods, you must [enable cluster checks][2]. This can be done by adding the `clusterchecks` configuration provider:
+**Note**: If the monitored endpoints are not backed by pods, you must [enable cluster checks][1]. This can be done by adding the `clusterchecks` configuration provider:
```

Comment:
```suggestion
**Note**: If the monitored endpoints are not backed by pods, you must [enable Cluster Checks][1]. This can be done by adding the `clusterchecks` configuration provider:
```